.github/workflows/main.yml

```diff
@@ -8,6 +8,42 @@ jobs:
     - uses: konradpabjan/move-labeled-or-milestoned-issue@v2.0
       with:
         action-token: "${{ secrets.MOVING_ISSUES_TOKEN }}"
-        project-url: "https://github.com/orgs/cerner/projects/3"
-        column-name: "To Do"
-        milestone-name: "Terra Application 2.0.0"
+        project-url: "https://github.com/orgs/cerner/projects/4"
+        column-name: "Planned"
+        milestone-name: "2020 Q2"
```

Comment:
Is the plan to rotate these as q2 finish?
so 2020Q2 will beocme 2022Q1?

Reply:
i think we could do that.  or we could take ownership of the actions repo and make it handle reg exes.  or we'll see if they do something with this.